objectivec/cxx_api.h

```diff
@@ -12,41 +12,26 @@
 
 // paths are different when building the Swift Package Manager package as the headers come from the iOS pod archive
 #ifdef SPM_BUILD
-
-#ifndef ENABLE_TRAINING_APIS
-#include "onnxruntime/onnxruntime_c_api.h"
-#include "onnxruntime/onnxruntime_cxx_api.h"
+#define HEADER_DIR "onnxruntime/"
```

Comment:
cool, glad to see the reduced duplication here. could we use a more specific name like `ORT_C_CXX_HEADER_DIR`?